### ISSUE TITLE
Menu with input doesn't when when menu is set to small

### ISSUE THREAD

**Issue by phillip-haydon** [Semantic-Org/Semantic-UI#173]
If you have a menu set to small:

`<nav class="ui small teal inverted menu">`

Then add a right hand side menu item like another drop down. The menu works perfectly fine.

If you have something on the right hand side + an input. The menu continues to work fine. If you only have an input on the right hand side, it breaks in weird ways.

Changing around fitted and such makes it weirder.

i.e

Normal
![image](https://f.cloud.github.com/assets/895629/1276488/f63f7e48-2e6d-11e3-8846-0d5cedfb4534.png)

Small
![image](https://f.cloud.github.com/assets/895629/1276491/0e055098-2e6e-11e3-8c97-6fbe063b5a58.png)

Removed right menu:
![image](https://f.cloud.github.com/assets/895629/1276492/20c227b0-2e6e-11e3-8ed5-10788481cd41.png)

Removing the `item` wrapper and making input/item the same:

i.e

```
      <div class="right menu">
          <div class="ui icon input item">
            <input type="text" placeholder="Search...">
            <i class="search link icon"></i>
          </div>
      </div>
```

![image](https://f.cloud.github.com/assets/895629/1276495/9805f04a-2e6e-11e3-9937-1cf736df538d.png)

Basically if I have any menu items on the right hand side I can't break it, the moment I have no menu item and just the input item, I can't get it working nicely.
